humilis_batch/meta.yaml.j2

```diff
@@ -98,3 +98,11 @@ meta:
         ec2_key_pair:
             description: The name of a SSH key to inject in the instances
             value:
+
+        has_launch_template:
```

Comment:
Isn't this parameter redundant with `user_data`? I.e. if no launch template configuration is provider (`user_data` is not provided) then `has_launch_template=no`.

Reply:
I see that [aws `launch_template` can have several information](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-launchtemplatedata.html) not just `user_data`. Right now, we only included `user_data` because that is the only parameter we are currently using. 

If we are to add a new information for `launch_template` then we will need to revise back to this approach. My thoughts is that this current approach allows us to add easily more informations that can be supplied for aws `launch_template`.

Let me know if you agree to this point. Else I can revise.

Reply:
If we later decide to use another `launch_template` configuration we can simply have `user_data OR whatever_other_configuration` as the condition. It's unlikely we'll be using too many params. IMO it's better than forcing the user of the plugin to specify two redundant properties.